.gitignore

```diff
@@ -15,3 +15,5 @@
 
 # Ignore Byebug command history file.
 .byebug_history
+
+.DS_Store
```

Comment:
It's possible to have a git `core.excludesfile` configuration that points to a global gitignore. We do this in our dotfiles, for example:

https://github.com/thoughtbot/dotfiles/blob/master/gitconfig#L16

Which points to a global one:

https://github.com/thoughtbot/dotfiles/blob/master/gitignore

And that's how our macOS users hide their `.DS_Store`.